draft-ietf-quic-http.md

```diff
@@ -88,28 +88,101 @@ code and issues list for this draft can be found at
 
 HTTP semantics are used for a broad range of services on the Internet. These
 semantics have commonly been used with two different TCP mappings, HTTP/1.1 and
-HTTP/2.  HTTP/2 introduced a framing and multiplexing layer to improve latency
-without modifying the transport layer.  However, TCP's lack of visibility into
-parallel requests in both mappings limited the possible performance gains.
+HTTP/2.  HTTP/3 supports the same semantics over a new transport protocol, QUIC.
+
+## Prior versions of HTTP
+
+HTTP/1.1 is a TCP mapping which uses whitespace-delimited text fields to convey
+HTTP messages.  While these exchanges are human-readable, the whitespace leads
```

Comment:
It isn't just the whitespace, but the use of whitespace for delineation makes some aspects of parsing and processing difficult.  You should be careful not to talk about shortcomings that this version doesn't address.

Reply:
Agree with @martinthomson 
Technically, whitespace is very easy to parse, and can be more efficient than variable length encoded frames. The real problem is the vague definition of what constitutes whitespace, when and where to expect it, and the different kinds of whitespace in existence. JSON has no such issues with whitespace.